docs/contributing-to-analyses/determining-requirements/index.md

```diff
@@ -6,8 +6,3 @@ When performing an analysis, there are two sets of requirements to be aware of a
 Tracking and reporting dependencies is important so that others can run your code and reproduce your results.
 
     - See the [Software Requirements](./determining-software-requirements.md) page for approaches to track and manage dependencies.
-
-1. **Compute requirements** refer to the computational resources, such as the number of CPUs or the amount of RAM, that are required for your analysis to run.
```

Comment:
Shouldn't the rest of this document change then, too? For example, there are no longer two sets of requirements being explained here.